recipes/pyDeprecate/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pyDeprecate" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c07e8ef5d9f40409183a952ddc288e7164c5d47644556077388c6b33c9aed545
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - deprecate
+
+about:
+  home: https://pypi.org/project/pyDeprecate/
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Collection of PyTorch native metrics for easy evaluating machine learning models'
```

Comment:
FYI, I think the dev url and summary are left over from copy/paste.

Reply:
yes, I was fixing it in the https://github.com/conda-forge/pydeprecate-feedstock/commit/4062fcd57803e256f43b34a48753dd2e1479b164#